api/src/transactions.rs

```diff
@@ -584,6 +633,7 @@ impl TransactionsApi {
         }
     }
 
+    /// Parses a single signed transaction
```

Comment:
s/Retrieves/Parses/ perhaps

Reply:
This doesn't retrieve it from anywhere though, it parses it from the input